src/app/CommandHandler.h

```diff
@@ -80,6 +129,21 @@ class CommandHandler : public Command
 
     CHIP_ERROR AddClusterSpecificFailure(const ConcreteCommandPath & aCommandPath, ClusterStatus aClusterStatus) override;
 
+    /**
+     * AddRef will increase the inner refcount of the CommandHandler.
+     *
+     * Users should use CommandHandler::Handle for management the lifespan of the CommandHandler.
+     * DefRef should be released in reasonable time, and Close() should only be called when the refcount reached 0.
+     */
+    void AddRef();
```

Comment:
Perhaps IncRef for symmetry with DecRef?

```suggestion
    void IncRef();
```

Reply:
I found BufferHandle is using AddRef instead of IncRef, not sure if this should be consistent.

Reply:
It's a minor thing and didn't need to block this PR.  But I think this makes more sense.

Thanks for the change.